lib/command/update/general_lastup_updater.rb

```diff
@@ -0,0 +1,106 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2013 whiteleaf. All rights reserved.
+#
+
+require_relative "../../narou/api"
+
+module Command
+  class Update < CommandBase
+    class GeneralLastupUpdater
+      def initialize(options)
+        @options = options
+        @database = Database.instance
+        @narou_novels = Hash.new { |h, k| h[k] = [] }
```

Comment:
__[reek]__
* [UncommunicativeVariableName] has the variable name 'h' [:ref](https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Variable-Name.md)
* [UncommunicativeVariableName] has the variable name 'k' [:ref](https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Variable-Name.md)

<!-- Comment by [SideCI](https://sideci.com) -->